src/v2/Apps/Settings/SettingsApp.tsx

```diff
@@ -39,6 +44,14 @@ const tabs = [
   },
 ]
 
+const tabs = sourceTabs.filter(tab => {
```

Comment:
Can we render tabs something like this?
```
{tabs.map(tab => {
  if (tab.name === "Alerts" && !getENV("ENABLE_SAVED_SEARCH")) {
    return null
  }
  return <RouteTab to={tab.url}>{tab.name}</RouteTab>
})}
```

So that we do not need to make extra iteration

Reply:
I don't think if using extra iteration will affect the performance in any way. And also in the future, when we get rid of feature flag, we will have to make fewer changes in the code (we can just remove `filter`)

Reply:
Just a suggestion, non blocking, feel free to keep it as it is